### DISCUSSION TITLE
Channel-specific system prompts

### DISCUSSION THREAD

**Discussion by mmiguel6288** [jakobdylanc/llmcord.py#82] [Ideas]
It would be nice to enable the config file to use different system prompts for different channels within a server.

**Comment by jakobdylanc** [comment]
Would be nice, but sacrifices simplicity. Most users wouldn't care about this functionality, and so adding it would only get in their way.

Not saying I'm against it. Maybe there's an elegant way to add it that I'm not thinking of.

**Comment by jakobdylanc** [comment]
https://github.com/jakobdylanc/llmcord/pull/83
